data/campaigns/World_Conquest/lua/game_mechanics/wocopedia/help.lua

```diff
@@ -53,19 +53,19 @@ function wesnoth.wml_actions.wc2_show_wocopedia(cfg)
 		local str_cat_feedback = _ "Feedback"
 		local str_des_feedback =
 			_ "<b>Feedback</b>:\n" ..
-			_ "For feedback plase either post in the Word conquest II thread in the official wesnoth forum https://r.wesnoth.org/t39651 or file an issue at github https://github.com/gfgtdf/World_Conquest_II/issues .\n\n" ..
+			_ "For feedback please either post in the World Conquest II thread in the official Wesnoth forum https://r.wesnoth.org/t39651 or file an issue at GitHub https://github.com/gfgtdf/World_Conquest_II/issues .\n\n" ..
```

Comment:
This should presumably be rewritten for mainline if it's still useful at all.

Reply:
Not within the scope of this PR.